ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -13,12 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *******************************************************************************/
-#include "ngraph_mark_for_clustering.h"
-#include "ngraph_api.h"  // TODO: Importing this first causes a compile error
-#include "ngraph_backend_manager.h"
-#include "ngraph_utils.h"
-#include "ngraph_version_utils.h"
+
 #include "tensorflow/core/graph/graph.h"
+
+#include "ngraph_bridge/ngraph_api.h"  // TODO: Importing this first causes a compile error
+#include "ngraph_bridge/ngraph_backend_manager.h"
+#include "ngraph_bridge/ngraph_mark_for_clustering.h"
+#include "ngraph_bridge/ngraph_utils.h"
+#include "ngraph_bridge/ngraph_version_utils.h"
```

Comment:
maybe this `TODO` is not relevant right now, but it does recommend importing `ngraph_mark_for_clustering` before `ngraph_api` (but i forget why)

Reply:
Need to fix this - the need for of include files indicate that there would be a problem in future. Need to fix this.

Reply:
We are importing it, before `ngraph_mark_for_clustering` and the compiler does not complain. I will remove the comment.